libenkf/applications/ert_tui/main.c

```diff
@@ -158,7 +158,10 @@ int main (int argc , char ** argv) {
     }
     enkf_welcome( model_config_file );
     {
-      enkf_main_type * enkf_main = enkf_main_bootstrap(model_config_file , true , true);
+      site_config_type * site_config = site_config_alloc_load_user_config(
+                                                            model_config_file
+                                                            );
+      enkf_main_type * enkf_main = enkf_main_alloc(model_config_file, site_config, true, true);
       enkf_main_run_workflows( enkf_main , workflow_list );
       enkf_tui_main_menu(enkf_main); 
       enkf_main_free(enkf_main);
```

Comment:
Now this scope manages the `site_config` instance - i.e. there should be a `site_config_free( site_config )` call.